### ISSUE TITLE
Generated GET requests include a `content-type` instead of an `accept` header [bug]

### ISSUE THREAD

**Issue by jamietanna** [ahmadnassri/httpsnippet#245]
When using the [Elements API Viewer](https://github.com/stoplightio/elements/), which uses this project, the [OpenAPI document here](https://github.com/co-cddo/federated-api-model/blob/54c24338ed37d3778bffbcc27ee96ccde8fb3a0e/schemas/v1alpha/openapi.yml) returns the following:

```
curl --request GET \
  --url /apis \
  --header 'Content-Type: application/json' \
  --header 'correlation-id: '
```

When we'd expect to see an `Accept` header here.

**Comment by dimitropoulos** [comment]
so I looked into this and I think I understand the problem.  It's to do with `application/vnd.uk.gov.api.v1alpha+json:` being a file schema ref on another file, if I had to guess.

To work on this, what we'd really need to see is the HAR format that Elements passes to HTTPSnippet, because I have a strong suspicion that _there_ is where the problem lies.  I believe that HTTPSnippet is not being passed the headers somewhere in the conversion from OpenAPI to HAR, because otherwise the headers would be present.

Ultimately, since HTTPSnippet doesn't deal with OpenAPI documents at all (only the [HAR format](https://en.wikipedia.org/wiki/HAR_(file_format))) I don't think we're going to be able to do anything about this.  I'm not sure who you would follow up on the stoplight side, maybe @philsturgeon?
